sdk/communication/azure-communication-identity/CHANGELOG.md

```diff
@@ -1,14 +1,12 @@
 # Release History
 
-## 1.5.0-beta.1 (Unreleased)
+## 1.4.4 (2023-03-08)
 
-### Features Added
-
-### Breaking Changes
+### Other Changes
 
-### Bugs Fixed
+#### Dependency Updates
 
-### Other Changes
+- Upgraded `azure-core` from `1.36.0` to version `1.37.0`.
 
```

Comment:
Also mention the azure-communication-common version upgrade (from 1.2.5 to 1.2.6 I guess)

Reply:
and identity one as well

Reply:
Done.